### PR TITLE
[BUG] Guide: Update the outdated  Link #915

### PR DESCRIPTION
**Pull request by NaincyKumariKnoldus**
Hey! I have tried to update these links. Please review my PR.
If any changes still needed do inform me I will update that.
Thanks!

### CODE REVIEW

File: static-channel-backup.md [hunk 1]
```diff
@@ -310,7 +310,7 @@ The `channel.backup` file is very small in size (<<1 MB) so even the smallest US
 
 ## Option 2: Remote backup preparations
 
-Follow this section if you want a remote backup. If you already set up a local backup, and don't want a remote backup, skip to the [next section](https://github.com/VajraOfIndra/RaspiBolt/edit/thumbrdive-scb-backup/static-channel-backup.md#test).
+Follow this section if you want a remote backup. If you already set up a local backup, and don't want a remote backup, skip to the [next section](https://raspibolt.org/static-channel-backup.html#option-2-remote-backup-preparations).
```
Comment (VajraOfIndra) [inline static-channel-backup.md:313]:
```suggestion
Follow this section if you want a remote backup. If you already set up a local backup, and don't want a remote backup, skip to the [next section](https://raspibolt.org/static-channel-backup.html#test-1).
```

The link should actually point to the next section whis is "Test".

**Review by VajraOfIndra — COMMENTED** [review]
Thanks for fixing the broken link, that's awesome!

Thinking of it, the link should actually point to the next section ("Test") as this is used to skip Option 2 altogether. I made a change suggestion below, but please double-check the link that it works.

**Review by martinatime — APPROVED** [review]

File: static-channel-backup.md (hunk 1 above)
Comment (martinatime) [reply static-channel-backup.md:313]: https://github.com/raspibolt/raspibolt/issues/915

Comment (NaincyKumariKnoldus) [reply static-channel-backup.md]: Hey @VajraOfIndra I have made the requested changes. Review that.

Comment (VajraOfIndra) [reply static-channel-backup.md]: LGTM! let's go to master :)

**Review by VajraOfIndra — APPROVED** [review]

**Pull request merged**